### PR TITLE
PUBDEV-6548: Specify certificate alias when starting H2O with SSL

### PR DESCRIPTION
**Pull request by michalkurka**

### CODE REVIEW

File: h2o-webserver-iface/src/main/java/water/webserver/iface/H2OHttpConfig.java [hunk 1]
```diff
@@ -28,4 +30,54 @@
 
   public String context_path;
 
+  @Override
+  public boolean equals(Object o) {
+    if (this == o) return true;
```
Comment (jakubhava) [inline h2o-webserver-iface/src/main/java/water/webserver/iface/H2OHttpConfig.java]: I'm sure that you thought about that, but is there any specific reason to implement the equals method, can't we use the default implementation?

**Comment by jakubhava** [comment]
Once this is in, we will also expose it in a form of spark option in sw https://0xdata.atlassian.net/browse/SW-1325

File: h2o-webserver-iface/src/main/java/water/webserver/iface/H2OHttpConfig.java (hunk 1 above)
Comment (michalkurka) [reply h2o-webserver-iface/src/main/java/water/webserver/iface/H2OHttpConfig.java]: Default implementation? Do you mean the equals from Object? That is just comparing the object reference.

Or do you mean equals generated at runtime by a utility function/library?

This code is generated by IntelliJ, not handwritten.

Comment (jakubhava) [reply h2o-webserver-iface/src/main/java/water/webserver/iface/H2OHttpConfig.java]: Cool! What do you think about mentioning that this was generated by IDE?

Comment (michalkurka) [reply h2o-webserver-iface/src/main/java/water/webserver/iface/H2OHttpConfig.java]: Sure, I will add!

**Pull request merged**